schemas/EconomicEvent.json

```diff
@@ -1,71 +1,107 @@
+        "toResourceInventoriedAs": {
+            "description": "Additional economic resource on the economic event when needed by the receiver. Used when a transfer or move, or sometimes other actions, requires explicitly identifying an economic resource on the receiving side.",
+            "$ref": "https://raw.githubusercontent.com/valueflows/valueflows/master/json-schemas/EconomicResource.json"
+        },
+        "resourceClassifiedAs": {
+            "description": "References a concept in a common taxonomy or other classification scheme for purposes of categorization or grouping.",
+            "type": "array",
+                "items": {
+                    "type": "string",
+                    "format": "uri"
+                }
+        },
+        "resourceConformsTo": {
+            "description": "The primary resource specification or definition of an existing or potential economic resource. A resource will have only one, as this specifies exactly what the resource is.",
+            "$ref": "https://raw.githubusercontent.com/valueflows/valueflows/master/json-schemas/ResourceSpecification.json"
         },
         "provider": {
             "description": "The economic agent from whom the economic event is initiated.",
-            "$ref": "https://raw.githubusercontent.com/valueflows/vf-json-schema/master/schemas/Agent.json"
+            "$ref": "https://raw.githubusercontent.com/valueflows/valueflows/master/json-schemas/Agent.json"
         },
         "receiver": {
             "description": "The economic agent whom the economic event is for.",
-            "$ref": "https://raw.githubusercontent.com/valueflows/vf-json-schema/master/schemas/Agent.json"
+            "$ref": "https://raw.githubusercontent.com/valueflows/valueflows/master/json-schemas/Agent.json"
+        },
+        "inScopeOf": {
+            "description": "Grouping around something to create a boundary or context, used for documenting, accounting, planning.",
+            "$ref": "https://raw.githubusercontent.com/valueflows/valueflows/master/json-schemas/Agent.json"
         },
         "hasBeginning": {
-            "description": "The starting date and optionally time of the economic event.",
+            "description": "Specific time marking the exact beginning of the economic event.",
             "type": "string",
             "format": "date-time"
         },
         "hasEnd": {
-            "description": "The ending date and optionally time of the economic event.",
+            "description": "Specific time marking the exact end of the economic event.",
             "type": "string",
             "format": "date-time"
         },
         "hasPointInTime": {
-            "description": "The economic event occurs at this date/time instant.",
+            "description": "Specific time marking the exact time at which the economic event occurred.",
             "type": "string",
             "format": "date-time"
         },
         "before": {
```

Comment:
What is the use of before and after in relation to beginning, end, and pointInTime?

Reply:
usually alternatives, but also might want a due date plus a planned interval, say - just gives different alternatives for exactness

Reply:
Possible to explain succinctly somewhere? Otherwise what shd implementers do?